### PR TITLE
Release 0.28.4

### PR DESCRIPTION
**Pull request by hugoesthere**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## 0.28.4 (2017-10-25)
+
+#### New Features
+
+* `exo-deploy`: allow traffic from bastion instances to RDS secruity groups
+
+#### Bug fixes
+
+* `exo-run`: fix ctrl-c shutdown bug introduced in previous release
```
Comment (charlierudolph) [inline CHANGELOG.md]: I don't think bug was every released.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## 0.28.4 (2017-10-25)
+
+#### New Features
+
+* `exo-deploy`: allow traffic from bastion instances to RDS secruity groups
```
Comment (charlierudolph) [inline CHANGELOG.md]: Were you able to test this with a deploy of EDX and be able to connect? Also thinking back on the implementation the 5432 port is specific to Postgres. Mysql for instance uses a different port but would also use RDS

Comment (hugoesthere) [reply CHANGELOG.md]: Good point. For this release I'll note that it only allows postgres traffic. I did a test deploy and let it go until the plan step. Then I read the plan to verify that the sg was being modified, but I didn't actually apply the deploy. Should I test that right now?

@alexdavid should we allow all traffic from bastion for each of the respective db ports?

Comment (charlierudolph) [reply CHANGELOG.md]: I don't think we necessarily need to support the other versions at the moment but at least build this in a way that we can add support for the other dbs down the line

Comment (hugoesthere) [reply CHANGELOG.md]: Yes it's certainly built in a way to allow support for other dbs, even right now when we're only supporting postgres

Comment (charlierudolph) [reply CHANGELOG.md]: Shouldn't the port be a variable though? Where 5432 is defined once as part of the postgres dependency and we would have a different one for mysql

Comment (alexdavid) [reply CHANGELOG.md]: Good point, didn't think about that. We honestly could just open up all communication to the bastion instance like we do for the ECS cluster.

Comment (charlierudolph) [reply CHANGELOG.md]: I like the simpler approach

File: src/cmd/deploy.go [hunk 3]
```diff
@@ -62,7 +62,7 @@ var deployCmd = &cobra.Command{
 			DeployServicesOnly: deployServicesFlag,
 
 			// git commit hash of the Terraform modules in Originate/exosphere we are using
-			TerraformModulesRef: "01891552",
+			TerraformModulesRef: "f95d4b0a",
```
Comment (charlierudolph) [inline src/cmd/deploy.go]: I think this needs to be updated.

**Review by charlierudolph — COMMENTED** [review]
Good to merge after updating the terraform module hash

**Pull request merged**